### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by liamfiddler**
Probably easiest if you go through the commits one by one instead of viewing all the changes at once.

I'm still on the fence about whether app-layout is a required dependency or not. It seems like overkill for this scale of app, but you'll probably want it if we use this as the basis for a future app.

### CODE REVIEW

File: polymer-punk-brew/src/punk-brew-app/punk-brew-app.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
     </iron-ajax>
 
 
-    <app-header effects="waterfall">
```
Comment (searleb) [inline polymer-punk-brew/src/punk-brew-app/punk-brew-app.html]: Weirdly, this effect worked even though it would error saying it was undefined 🤔

File: polymer-punk-brew/src/punk-brew-app/punk-brew-app.html [hunk 2]
```diff
@@ -75,18 +73,6 @@
...
         };
       }
-
-      static get observers() {
-        return [
-          '_routePageChanged(routeData.page, subrouteData.id)',
-        ];
-      }
-
-      _routePageChanged(page, id) {
```
Comment (searleb) [inline polymer-punk-brew/src/punk-brew-app/punk-brew-app.html:86]: @liamfiddler Why was this removed?

File: polymer-punk-brew/src/punk-brew-app/punk-beer-details.html [hunk 3]
```diff
@@ -50,8 +50,6 @@
           </template>
         </ul>
 
-        <a href="/beer/[[beer.id]]">More</a>
```
Comment (searleb) [inline polymer-punk-brew/src/punk-brew-app/punk-beer-details.html:49]: @liamfiddler How do you click through to the details page without the link?

File: polymer-punk-brew/src/punk-brew-app/punk-brew-app.html (hunk 2 above)
Comment (liamfiddler) [reply polymer-punk-brew/src/punk-brew-app/punk-brew-app.html:86]: If the value is `undefined` it will automatically use the value specified in the `fallback-selection` attribute on iron-pages. Which is currently set to "home".

https://www.webcomponents.org/element/PolymerElements/iron-pages/elements/iron-pages#property-fallbackSelection

File: polymer-punk-brew/src/punk-brew-app/punk-beer-details.html (hunk 3 above)
Comment (liamfiddler) [reply polymer-punk-brew/src/punk-brew-app/punk-beer-details.html:49]: This link was inside the details component. You're already on that page! 🤠
